egs/wsj/s5/steps/cleanup/append_eval_to_ctm.py

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/env python
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
+from __future__ import print_function
+import sys, operator, argparse
+
+# Append to the CTM the Levenshtein alignment of 'hypothesis' and 'reference':
+# (i.e. the output of 'align-text' post-processed by 'wer_per_utt_details.pl')
+
+# The tags in the processed Levenshtein alignment are:
+#  'C' = correct
+#  'S' = substitution
+#  'D' = deletion
+#  'I' = insertion
+#  'EC' = silence and neighboring words are both correct
```

Comment:
OK, the <eps> in the output of nbest-to-ctm do correspond to optional silences.  If that's what you refer to, better use the letter 'S' not 'E'.  There is still a lot here that is unclear to me-- perhaps when you add an example to the  comment, it will become more clear.